### PR TITLE
Feature/tf ci

### PR DESCRIPTION
**Pull request by atrakic**
Related to #161. Adds a basic CI for the /modules. 
Note: I had to update some of the modules in order to pass CI.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+
+env:
+  TF_VERSION: 1.4.1
+  TF_IN_AUTOMATION: true
+  TF_PLUGIN_CACHE_DIR: ${{ github.workspace }}/.terraform.d/plugin-cache
+
+jobs:
+    terraform:
+        name: 'Terraform CI'
+        runs-on: ubuntu-latest
```
Comment (alexott) [inline .github/workflows/ci.yml:20]: We need to run it on Databricks-managed runners, like this:

https://github.com/databricks/databricks-jdbc-dev/blob/main/.github/workflows/prCheck.yml#L18

**Review by alexott — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (atrakic) [reply .github/workflows/ci.yml:20]: Thx, but I get "Access forbidden from this IP address" when I follow the link. 
Any chance allowing running this ci job on gh hosted runners?

Comment (alexott) [reply .github/workflows/ci.yml:20]: I've updated the link

Comment (atrakic) [reply .github/workflows/ci.yml:20]: Not sure if this would work for prs from non gh org members. I'm closing this one for now.

**Comment by atrakic** [comment]
Not applicable due to gh runner restrictions.

**Pull request closed without merging**